src/components/CreateOptions.tsx

```diff
@@ -0,0 +1,267 @@
+import { useState } from "react";
+import Editor from "@monaco-editor/react";
+
+interface Props {
+  activeOption: string | null;
+  setActiveOption: (option: string) => void;
+  setHasUnsavedChanges: (value: boolean) => void;
+  onCancel: () => void;
+}
+
+const CreateOptions = ({
+  activeOption,
+  setActiveOption,
+  setHasUnsavedChanges,
+  onCancel,
+}: Props) => {
+  const [yamlContent, setYamlContent] = useState("");
```

Comment:
```suggestion
  // eslint-disable-next-line no-unused-vars
  const [yamlContent, setYamlContent] = useState("");
```

you can add the eslint comment for this unused vars

Reply:
ok